Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>An issue I found here is that the account explorer notebooklet might return a selection list if it matches multiple accounts.</p><p>This doesn't get displayed but then you call notebooklet functions that say "Please use select an account before using this method."</p><p>I think this might be because the CyberSecuritySoc being a bit crap - there are a ton of alerts for JeffL but no SigninLogs or other activity!!!</p><p> </p><p>Not sure what the right thing to do here is - the result will have an account_selector property set to an nbwidgets.SelectItem.</p><p>You could test for that and maybe warn that multiple accounts matched and you're displaying the first one.</p><pre class="ql-syntax" spellcheck="false">if acc_nb.account_selector is not None:
     print(warning..)
     acc_nb.account_selector
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>